src/main/java/no/unit/nva/cristin/projects/model/nva/NvaOrganization.java

```diff
@@ -0,0 +1,43 @@
+package no.unit.nva.cristin.projects.model.nva;
+
+import static no.unit.nva.cristin.projects.JsonPropertyNames.ID;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.NAME;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.TYPE;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import java.net.URI;
+import java.util.Map;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+@JsonPropertyOrder({ID, TYPE, NAME})
+public class NvaOrganization {
+
+    private URI id;
+    private String type;
+    @JsonPropertyOrder(alphabetic = true)
+    private Map<String, String> name;
```

Comment:
I'm tempted by something called LanguageValue(String language, String name), but we should maybe think more about this.